src/Command.js

```diff
@@ -166,7 +174,10 @@ export default class Command {
   }
 
   runValidations() {
-    if (!GitUtilities.isInitialized(this.repository.rootPath)) {
+    const opts = Object.assign({}, this.execOpts, {
+      cwd: this.repository.rootPath
```

Comment:
Actually, `GitUtilities.isInitialized()` should _already_ be passed `this.execOpts`, since `this.repository.rootPath === this._execOpts.cwd` 😭 

(so, uh, that's a gaping hole in our unit tests that I caused)